pernosco-submit

```diff
@@ -718,6 +719,21 @@ def get_metadata_for_upload(public_key, signature, pernosco_user, pernosco_group
         metadata += ",%s"%os.environ['PERNOSCO_EXTRA_METADATA']
     return metadata
 
+def append_crypto_data_to_package(package_file_name, crypto_data):
+    """
+    Append cryptographic data required to upload the package at a later time
+    to the package.
+
+        Parameters:
+            package_file_name (str): Name of the package file to be uploaded later
+            crypto_data (dict): Cryptographic data required to upload the package
+    """
+    print("Appending cryptographic data required for upload to %s"%package_file_name)
+    crypto_data_json = json.dumps(crypto_data)
+    with open(package_file_name, "a") as package_file:
+        print("%s"%crypto_data_json, file=package_file)
```

Comment:
I think the `"%s"%` is unnecessary?

Reply:
You are right. Just a habit I have from secure C programming. :)